### ISSUE TITLE
NTR: Cold agglutinin test positive (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8790]
New term request Cold agglutinin test positive (segal_210723213032): Cause red blood cells to clump together at low temperatures  UMLS: 0235943

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000386
- label: Positive cold agglutinin test
- parents: [Autoimmune antibody positivity (HP:0030057)](https://hpo.jax.org/app/browse/term/HP:0030057)
- pmid: PMID:34285751
- def: Cold agglutinin are erythrocyte antibodies which possess the property of agglutinating red blood cells at temperatures of below 37 degrees C, this phenomenon is reversible after heating.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8790
- synonyms:

**Comment by MaudDLVT** [comment]
Hello Peter @pnrobinson ,
May I request to change the name of this autoantibody, as it stands in the Autoimmune antibody positivity category (not a test category)?
May I suggest: IgM cold agglutinin autoantibody positivity
Synonyms: anti-I antibody, Cold-reactive antibody
Ref: PMID: 34595449, PMID: 35917437, PMID: 33435309
Many thanks,
Maud
